locust/main.py

```diff
@@ -100,6 +100,26 @@ def main():
             user_classes[key] = value
             available_user_classes[key] = value
 
+    from .stats import PERCENTILES_TO_CHART
```

Comment:
group this import with the one on line 16

Reply:
Thank you for the comment. This is same situation as I explained html.py

Importing PERCENTILES_TO_CHART at 16 does not work. What I mean by that,
stats. PERCENTILES_TO_CHART default is [0.95, 0.5] as list in stats.py. When passing new value from locust execution file like stats. PERCENTILES_TO_CHART = [0.8,0.3] as list for instance, when PERCENTILES_TO_CHART was read from main.py file, default [0.95, 0.5] was used not new value [0.8,0.3]. But importing PERCENTILES_TO_CHART at line 78 which is right before PERCENTILES_TO_CHART is used, new value [0.8,0.3] is used by main.py. I think this is caused by pass by reference etc.. Actually, passing new value as PERCENTILES_TO_CHART[0] = 0.8 works, even if importing PERCENTILES_TO_CHART at line 6. I would like to suggest to use PERCENTILES_TO_CHART_1 and PERCENTILES_TO_CHART_2 in stats.py not as list. This should work.I hope this explanation makes sense to you. If you have any suggestion, let me know.

Reply:
import the whole stats module and access it as `stats.PERCENTILES_TO_CHART` and it should work!

Reply:
Like here: https://github.com/locustio/locust/blob/master/locust/main.py#L245

Reply:
Yes, that worked! thank you.